astropy/units/format/generic.py

```diff
@@ -466,6 +455,30 @@ def _parse_unit(cls, s, detailed_exception=True):
         else:
             raise ValueError()
 
+    _unit_symbols = {
+        '%': 'percent',
+        '\N{PRIME}': 'arcmin',
+        '\N{DOUBLE PRIME}': 'arcsec',
+        '\N{MODIFIER LETTER SMALL H}': 'hourangle',
+        'e\N{SUPERSCRIPT MINUS}': 'electron',
```

Comment:
how is `e⁻` a unit?

Reply:
Predates me, but I think meant to be CCD counts or so, https://github.com/astropy/astropy/blob/d33db763dcdec85c2904667b9787b46d5cf4865e/astropy/units/astrophys.py#L121-L122

Reply:
Yes, I used it at some point to go from photons --> electrons --> adu in an ETC.